man/coqtop.1

```diff
@@ -110,7 +110,7 @@ print Coq version and exit
 
 .TP
 .B \-q
-skip loading of rcfile
+skip loading of rcfile (resource file if any)
```

Comment:
I would write skip loading of rcfile (resource file) if any

Reply:
Right, changed.

No opinion otherwise on the rc file and on whether `Add LoadPath` is "good" or "bad".